samples/nrf9160/aws_fota/src/main.c

```diff
@@ -490,14 +493,21 @@ static void aws_fota_cb_handler(struct aws_fota_event *fota_evt)
 		err = lte_lc_offline();
 		if (err) {
 			printk("Error turning off the LTE link\n");
+			break;
 		}
+		link_offline = true;
 		break;
 
 	case AWS_FOTA_EVT_ERASE_DONE:
-		printk("AWS_FOTA_EVT_ERASE_DONE, reconnecting the LTE link\n");
-		err = lte_lc_connect();
-		if (err) {
-			printk("Error reconnecting the LTE link\n");
+		printk("AWS_FOTA_EVT_ERASE_DONE\n");
+		if (link_offline) {
+			printk("Reconnecting the LTE link\n");
+			err = lte_lc_connect();
```

Comment:
Same here

Reply:
Not sure if I added it like you wanted but tried :)